packages/react-charting/src/components/AreaChart/__snapshots__/AreaChart.test.tsx.snap

```diff
@@ -33,6 +33,7 @@ exports[`AreaChart - mouse events Should render callout correctly on mouseover 1
           overflow: auto;
         }
     data-focuszone-id="FocusZone8"
+    data-tabster="{\\"uncontrolled\\": {}}"
```

Comment:
Changes in react charting package look unrelated. These should be reverted.

Reply:
This is not a change in react-charting. This just updates the snapshots that have been changed because of the FocusZone change.

Reply:
I saw a change in data of the snapshot that looked unrelated. I can't see the change anymore so assuming that is resolved now.